core/converters.go

```diff
@@ -4,6 +4,9 @@ import (
 	"encoding/base64"
```

Comment:
Maybe move the converters in a subpackage core/converters? We will add other converters there too (uint64ByteSliceConverter)

Reply:
not in this pull request. task added.

Reply:
ok